hnn_core/tests/test_gui.py

```diff
@@ -1109,3 +1109,12 @@ def test_update_nested_dict():
     updated = _update_nested_dict(original, has_nulls)
     expected = has_nulls
     assert updated == expected
+
+
+def test_delete_single_drive(setup_gui):
+    """Deleting a single drive."""
+    gui = setup_gui
+    assert len(gui.drive_accordion.children) == 6
+
+    gui._simulate_delete_single_drive(0)
+    assert len(gui.drive_accordion.children) == 5
```

Comment:
possible to check that the correct drive was deleted?

Reply:
Updated this to also have explicit checks for titles. Turns out titles weren't being updated correctly, so thanks for bringing it up!